app/views/participants/fields/_family_history.html.erb

```diff
@@ -1,6 +1,9 @@
 <%= f.inputs :name => "Family History Information" do -%>
 
-    <%= f.input :first_generation, :as => :select, :label => "Did parents graduate from college?", :collection => { "No (student is first generation to college)" => true, "Yes (student is NOT first generation to college)" => false } %>
+    <%= f.input :first_generation, :as => :select, :collection => { "No (student is first generation to college)" => true, "Yes (student is NOT first generation to college)" => false } unless @current_user.person_type == "Participant" %>
+
+    <%= f.input :first_generation, :as => :select, :collection => { "No (I am first generation to college)" => true, "Yes (I am NOT first generation to college)" => false } unless @current_user.person_type != "Participant" %>
```

Comment:
@mattharris5 @mitchellharper12 Is there a better way to display the text inside collection for different audiences?